jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -209,7 +209,7 @@ public ServletChannelState getServletRequestState()
         return _state;
     }
 
-    public long getBytesWritten()
+    private long getBytesWritten()
```

Comment:
Is it okay to change this to private now?

Reply:
ServletChannel.getBytesWritten() is used by metrics libraries, it should remain public.

Reply:
@joakime `ServletChannel.getBytesWritten()` should not be used by anyone, in general the whole `ServletChannel` class should be internal.
We have many other public ways to obtain the number of bytes written, so I think it's fine to make it private as it should have not been public in the first place.